### ISSUE TITLE
Some issues about wgan-gp with acgan condition  and conditional batchnorm 

### ISSUE THREAD

**Issue by weihancug** [igul222/improved_wgan_training#38]
I'm interested in the wgan-gp with adding AC-GAN conditioning.  But there is not introduction about this model in the [paper](https://arxiv.org/abs/1704.00028).   The code is [gan_cifar_resnet.py](https://github.com/igul222/improved_wgan_training/blob/master/gan_cifar_resnet.py) . In this model, a conditional batchnorm is used. But this explanation in the code is too small for me to get a better understanding. So can you supply the some information or paper about conditional batchnorm @igul222 .  In additional, I want it to generate 128x128 pixels images. Can you give me some advises?

Looking forward to your reply.

**Comment by xunhuang1995** [comment]
See this paper for some background about conditional batch norm: 
https://arxiv.org/abs/1610.07629

**Comment by gdshen** [comment]
If we add an acgan branch for the critic, can we use batch norm for the classifier of labels?
